etcdutl/etcdutl/check_command.go

```diff
@@ -0,0 +1,112 @@
+// Copyright 2024 The etcd Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package etcdutl
+
+import (
+	"errors"
+	"fmt"
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+
+	"go.etcd.io/etcd/pkg/v3/cobrautl"
+	"go.etcd.io/etcd/server/v3/etcdserver/api/membership"
+	"go.etcd.io/etcd/server/v3/etcdserver/api/snap"
+	"go.etcd.io/etcd/server/v3/etcdserver/api/v2store"
+	"go.etcd.io/etcd/server/v3/wal"
+)
+
+// NewCheckCommand returns the cobra command for "check".
+func NewCheckCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "check <subcommand>",
+		Short: "commands for checking properties",
+	}
+	cmd.AddCommand(NewCheckV2StoreCommand())
+	return cmd
+}
+
+var (
+	argCheckV2StoreDataDir string
+)
+
+// NewCheckV2StoreCommand returns the cobra command for "check v2store".
+func NewCheckV2StoreCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "v2store",
+		Short: "Check custom content in v2store",
+		Run:   checkV2StoreRunFunc,
+	}
+	cmd.Flags().StringVar(&argCheckV2StoreDataDir, "data-dir", "", "Required. A data directory not in use by etcd.")
+	cmd.MarkFlagRequired("data-dir")
+	return cmd
+}
+
+func checkV2StoreRunFunc(_ *cobra.Command, _ []string) {
+	err := checkV2StoreDataDir(argCheckV2StoreDataDir)
+	if err != nil {
+		cobrautl.ExitWithError(cobrautl.ExitError, err)
+	}
+}
+
+func checkV2StoreDataDir(dataDir string) error {
+	var (
+		storeClusterPrefix = "/0"
+		storeKeysPrefix    = "/1"
```

Comment:
We can reuse the existing definitions,
https://github.com/etcd-io/etcd/blob/762e93874f23213c3b3d918ba82ce2803bee3d46/server/etcdserver/server.go#L83-L84

Reply:
Updated.